### PR TITLE
Create basic authentication topic.

### PR DESCRIPTION
**Pull request by KoryKessel-Mirantis**

### CODE REVIEW

**Review by nwneisen — APPROVED** [review]
LGTM

**Review by brianbancroft — APPROVED** [review]

File: content/docs/configuration/authentication/basic-authentication.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+{{< /callout >}}
+
+An admin user can create users, change user passwords, and delete users from
+the MKE system.
+
+## Create users
+
+1. Log in to the MKE Dashboard as an administrator.
+
+2. Navigate to **Access Control** --> **Users** to access the **user** page.
```
Comment (MagdaDziadosz) [inline content/docs/configuration/authentication/basic-authentication.md:21]:
```suggestion
2. Navigate to **Access Control** --> **Users**.
```

File: content/docs/configuration/authentication/basic-authentication.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+3. Click the **Create User** button at the far right to access the **User Creation** page.
+
+4. Enter the perinent information into the following fields:
+
+   * **username**
```
[the remark this answers is not in the document]
Comment (KoryKessel-Mirantis) [reply content/docs/configuration/authentication/basic-authentication.md:29]: Not true. It is the **name** field that is optional.

image

File: content/docs/configuration/authentication/basic-authentication.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+## Delete users
+
+1. Log in to the MKE Dashboard as an administrator.
+
+2. Navigate to **Access Control** --> **Users** to access the **user** page.
+
+3. Tick the checkbox to the left of the username for the user you want to
+   delete. You can tick multiple boxes to delete multiple users at the same time.
+
+4. Click the **actions** dropdown at the far top right of the page and select
```
[the remark this answers is not in the document]
Comment (KoryKessel-Mirantis) [reply content/docs/configuration/authentication/basic-authentication.md:67]: Excellent point. I did not follow through on the checkbox delete operation the way I did with the alternate.

**Review by MagdaDziadosz — APPROVED** [review]

**Pull request merged**